pkg/brokerapi/openservicebroker/open_service_broker_client_test.go

```diff
@@ -202,11 +210,120 @@ func TestDeprovisionInstanceAcceptedFailureAsynchronous(t *testing.T) {
 		AcceptsIncomplete: true,
 	}
 
-	err := c.DeleteServiceInstance("1", &req)
+	err := c.DeleteServiceInstance(testServiceInstanceID, &req)
 	switch {
 	case err == nil:
 		t.Errorf("Expected '%v'", errFailedState)
 	case err != errFailedState:
 		t.Errorf("Expected '%v', got '%v'", errFailedState, err)
 	}
 }
+
+func TestBindOk(t *testing.T) {
+	fbs, fakeBroker := setup()
+	defer fbs.Stop()
+
+	c := NewClient(testBrokerName, fakeBroker.Spec.URL, "", "")
+
+	fbs.SetResponseStatus(http.StatusOK)
+	sent := &brokerapi.BindingRequest{}
+	if _, err := c.CreateServiceBinding(testServiceInstanceID, testServiceBindingID, sent); err != nil {
+		t.Error(err.Error())
```

Comment:
just wondering, is it worth continuing in cases like this? or should it be t.Fatal() ?